test/ui/ComponentOptionsTest.ts

```diff
@@ -232,7 +232,7 @@ export function ComponentOptionsTest() {
               required: true
             })
           };
-          expect(() => { ComponentOptions.initComponentOptions(elem, { options, ID: 'fooID' }); }).toThrow();
+          expect(() => { ComponentOptions.initComponentOptions(elem, { options, ID: 'fooID' }); }).not.toThrow();
```

Comment:
Is it me or the name of the tests in this class are not very informative? They don't mention what is expected.